server/src/main/java/org/opensearch/cluster/routing/RoutingNodes.java

```diff
@@ -227,6 +237,14 @@ public int getOutgoingRecoveries(String nodeId) {
         return recoveriesPerNode.getOrDefault(nodeId, Recoveries.EMPTY).getOutgoing();
     }
 
+    public int getInitialIncomingRecoveries(String nodeId) {
+        return initialRecoveriesPerNode.getOrDefault(nodeId, Recoveries.EMPTY).getIncoming();
+    }
+
+    public int getInitialOutgoingRecoveries(String nodeId) {
+        return initialRecoveriesPerNode.getOrDefault(nodeId, Recoveries.EMPTY).getOutgoing();
+    }
+
```

Comment:
Lets add java docs as it maybe confusing to users that initial recoveries are local to the node